webservices/rest.py

```diff
@@ -201,6 +202,43 @@ def add_caching_headers(response):
     return response
 
 
+@app.after_request
+def add_secure_headers(response):
+    """
+    Add secure headers to each response.
+    The 'unsafe-inline' Content Security Policy (CSP) setting is
+    needed for Swagger docs (see https://github.com/swagger-api/swagger-ui/issues/3370)
+    """
+
+    headers = {
+        "X-Content-Type-Options": "nosniff",
+        "X-Frame-Options": "Deny",
+        "X-XSS-Protection": "1; mode=block",
+    }
+    content_security_policy = {
+        "default-src": "'self' *.fec.gov *.app.cloud.gov",
+        "img-src": "'self' data:",
+        "script-src": "'self' 'unsafe-inline'",
+        "style-src": "'self' https://fonts.googleapis.com 'unsafe-inline'",
+        "font-src": "'self' https://fonts.gstatic.com data:",
+        "connect-src": "*.fec.gov *.cloud.gov",
+        "object-src": "'none'",
+        "report-uri": "/report-csp-violation/",
```

Comment:
It appears that `report-uri` directive is deprecated. Only few browser support this directive. Could you replace this directive with the new `report-to` or add a forward compatability to redirect all `report-uri` to `report-to` as described here:
 https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/report-uri

Reply:
@pkfec thanks for taking a look! I implemented `report-uri` because it's still supported by more browsers than `report-to`, but I'll go ahead and add `report-to` now.